app/sessions/sessions_manager.go

```diff
@@ -69,6 +75,30 @@ func (sm *SessionsManager) initWatchers() {
 	go sm.consumeInstanceStopped(instanceStopped)
 }
 
+func (sm *SessionsManager) initContainerManagementStart() {
+	output, err := sm.provider.GetManagedContainers()
```

Comment:
Instead of `GetManagedContainers`, you should use `sm.groups` which is already populated with the discovered instances

Reply:
Okay using sm.groups does make sense and simplified things. In that case we don't need to update the interface, and I can remove the "GetManagedContainers".

I think I will need to flatten a copy of the sm.groups, as this is a map with the group name as the key. And I believe the status API should be flat with the group as a property of the instance, rather than being grouped by groupname (which the API consumer can do on the client side)